rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/util/api.py

```diff
@@ -34,29 +33,23 @@ def call_api(self, path: str, request_type: str, params: dict = None, body: dict
             )
             if response.status_code not in [200, 201, 202]:
                 raise PluginException(
-                    cause=f"Failed to get a valid response from InsightVM at endpoint {api_url}",
-                    assistance=f"Response was {response.request.body}",
+                    cause=f"Failed to get a valid response from InsightVM at endpoint '{api_url}'",
+                    assistance=f"Response was {response.request.body}.",
                     data=response.status_code,
                 )
             if response.text == "":
-                return response.status_code
+                return {"status_code": response.status_code}
```

Comment:
Can you return a tuple here to make the expected return consistent? The issue with this update is that while a dict is always being returned, there's little consistency in the keys (now status_code has to be checked for). 
Something like the following may work great here:
```
return (response.status_code, None)
```
and then for the successful call condition below,
```
return (response.status_code, response.json())
```
The type annotation for the function would be updated like `-> (int, Optional[dict])` and an import of `from typing import Optional` will be needed.

At the call site, it would look something like the following...
```
response = call_api(...)
if response[0] in range(200, 299) and response[1] is not None:
    # do something with a success
else:
    # do something with a failure response
```